Composer/packages/electron-server/src/electronWindow.ts

```diff
@@ -21,20 +21,24 @@ export default class ElectronWindow {
 
   private constructor() {
     // Create the browser window.
+    const { width, height } = screen.getPrimaryDisplay().workAreaSize;
     const browserWindowOptions: Electron.BrowserWindowConstructorOptions = {
-      width: 800,
-      height: 600,
+      width: width * 0.75,
+      height: height * 0.9,
       webPreferences: {
         nodeIntegration: false,
+        preload: join(__dirname, 'preload.js'),
       },
       show: false,
+      title: `Bot Framework Composer (v${app.getVersion()})`,
     };
     if (process.platform === 'linux' && !isDevelopment) {
       // workaround for broken .AppImage icons since electron-builder@21.0.1 removed .AppImage desktop integration
       // (https://github.com/electron-userland/electron-builder/releases/tag/v21.0.1)
       browserWindowOptions.icon = join(getUnpackedAsarPath(), 'resources/composerIcon_1024x1024.png');
     }
     this._currentBrowserWindow = new BrowserWindow(browserWindowOptions);
+    this._currentBrowserWindow.on('page-title-updated', ev => ev.preventDefault());
```

Comment:
What's this for?

Reply:
In the initialization code for the main browser window, I explicitly set the title. Later on in the app lifecycle, there are times when Electron might try to update the browser title (this happens on the initial load of `index.html` and Electron will use whatever is in the `<head><title></title><head>` tag -- which doesn't include the version) and overwrite this explicitly set title.

This line prevents that behavior and leaves the explicitly set title.

Reply:
Ah ok. Would you mind just adding a brief comment stating that this prevents the overwriting of the window title?

Reply:
done!